crates/doc/src/combine/memtable.rs

```diff
@@ -224,15 +224,15 @@ impl MemTable {
     pub fn spill<F: io::Read + io::Write + io::Seek>(
         self,
         writer: &mut SpillWriter<F>,
-        chunk_target_size: ops::Range<usize>,
+        chunk_target_size: usize,
```

Comment:
The comment above here mentions a range, and could be updated since this isn't a range anymore.